PullUpController/PullUpController.swift

```diff
@@ -253,6 +253,7 @@ open class PullUpController: UIViewController {
             withDuration: animationDuration,
             animations: { [weak self] in
                 self?.parent?.view.layoutIfNeeded()
+            }, completion: { [weak self] b in
```

Comment:
Could you please move `completion..` to the next line? Also what is `b`? If you don't need the variable use the placeholder `_`.